packages/next-urql/README.md

```diff
@@ -213,6 +213,38 @@ let default = (withUrqlClient(. clientOptions))(. make);
 
 The great part about writing thin bindings like this is that they are zero cost – in fact, the above bindings get totally compiled away by BuckleScript, so you get the full benefits of type safety with absolutely zero runtime cost!
 
+### Usage with SSR mode
```

Comment:
"SSR mode" implies that Next has some sort of special "mode" (like Preview mode) for server-side rendering. I think this could better be described as:

```suggestion
### Restricting Data Fetching to the Server
```

From what I can understand, the goal of this change is to allow folks to be able to restrict data fetching with `urql` to the server rather than opting into combined client / server fetching.

Reply:
Yes, you understood it correctly.